plugins/catalog-backend/src/next/database/DefaultProcessingDatabase.ts

```diff
@@ -235,7 +235,7 @@ export class DefaultProcessingDatabase implements ProcessingDatabase {
               .withRecursive('ancestors', function ancestors(outer) {
                 return outer
                   .select({
-                    root_id: tx.raw('NULL', []),
+                    root_id: tx.raw('CAST(NULL as INT)', []),
```

Comment:
Only question is, should it be `as INTEGER NULL)` since it's indeed nullable, but not sure it's needed

Reply:
Good question!
I have confirmed that it works as intended with this code so let's merge it to fix deletions for now. We can take another look next week to make it better :)